CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ _None_
 
 ### Internal Changes
 
-_None_
+- Update the CI image used to build this project to use `xcode-14.1`. [#431]
```

Comment:
Does this set a precedent for adding an entry here every time we bump the CI image? I fear it might amount to more noise than signal. What do you think?

Reply:
Yeah, you might be right. I'm ok to remove it.

[EDIT] Oh, forgot I had auto-merge enabled on this PR so it's already merged… oh well.